core/lib/presentation/views/html_viewer/html_content_viewer_on_web_widget.dart

```diff
@@ -137,6 +144,24 @@ class _HtmlContentViewerOnWebState extends State<HtmlContentViewerOnWeb> {
     });
   }
 
+  void _handleIframeOnScrollChangedListener(
```

Comment:
to prevent handle multiple times on one even, we should return true or false, base on the `data`. If we handle scroll -> return true, else return false. Base on the result, we can break the invoke function to do other logic.

Reply:
Updated